dlt/helpers/ibis.py

```diff
@@ -109,11 +109,14 @@ def _ignore_hstore(conn: Any, name: Any) -> Any:
         con = ibis.snowflake.connect(
             schema=dataset_name, **sn_credentials, create_object_udfs=False
         )
-    elif issubclass(destination.spec, MsSqlClientConfiguration):
+    elif issubclass(destination.spec, MsSqlClientConfiguration) and not issubclass(
+        destination.spec, SynapseClientConfiguration
+    ):
         from dlt.destinations.impl.mssql.mssql import MsSqlJobClient
 
         assert isinstance(client, MsSqlJobClient)
         ms_credentials = client.config.credentials.to_native_representation()
+        ms_credentials = ms_credentials.replace("synapse://", "mssql://")
```

Comment:
same here? seems like you're replacing any unexpected credential prefixes that would still be thre

Reply:
ah, yes, this can go, thanks!